cypress/integration/ambianic-tests/timeline.spec.js

```diff
@@ -1,29 +1,29 @@
 /// <reference types="cypress" />
 
 context('Timeline', () => {
-    before(() => {
```

Comment:
@vickywane please avoid committing changes that are not directly related to the PR. It adds time and confusion to the review process.

Reply:
The changes here are the same. I only linted the file before pushing it.

I have created [this feature request ](https://github.com/ambianic/ambianic-ui/issues/678)to provide a solution to the cause of the unneeded change when a linting is done.